games/static/games/proto.js

```diff
@@ -338,6 +344,8 @@ class Board {
                     this.updateAlienHitAt(nextRocketLocation)
                 }
             }
+            // After checking and preparing all conditions we put down the rocket in the 
+            // next location up the board
```

Comment:
its a bit confusing because in the next line you use `rocketLocation`

Reply:
updated the comment to explain why